sched/tls/tls_initinfo.c

```diff
@@ -22,9 +22,13 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/config.h>
+
 #include <assert.h>
 #include <errno.h>
 
+#include <nuttx/cancelpt.h>
```

Comment:
why need change this file

Reply:
Not needed anymore, yes. It was because of the initializatin of tl_cpstate/count but I removed them